src/zjs_ble.c

```diff
@@ -1282,10 +1278,12 @@ jerry_value_t zjs_ble_init()
     // bt events are called from the FIBER context, since we can't call
     // zjs_trigger_event() directly, we need to register a c callback which
     // the C callback will call zjs_trigger_event()
-    ble_conn->ready_cb.id = zjs_add_c_callback(ble_conn, zjs_ble_ready_c_callback);
-    ble_conn->connected_cb.id = zjs_add_c_callback(ble_conn, zjs_ble_connected_c_callback);
-    ble_conn->disconnected_cb.id = zjs_add_c_callback(ble_conn, zjs_ble_disconnected_c_callback);
-    ble_conn->ble_obj = ble_obj;
+    ble_conn.ready_cb_id = zjs_add_c_callback(&ble_conn,
+                                              zjs_ble_ready_c_callback);
+    ble_conn.connected_cb_id = zjs_add_c_callback(&ble_conn,
+                                                  zjs_ble_connected_c_callback);
+    ble_conn.disconnected_cb_id = zjs_add_c_callback(&ble_conn, zjs_ble_disconnected_c_callback);
```

Comment:
That one didn't fit in 80 chars, even after doing that, is why I didn't. But yeah I'll do something.